frontend/chat-plugin/src/App.module.scss

```diff
@@ -5,7 +5,7 @@
   right: 0;
   bottom: 0;
   max-height: 500px;
-  max-width: 300px;
+  max-width: 320px;
```

Comment:
I hope it is okay to make the chat plugin 20px wider. The google rich card needs that space. Otherwise I can also tweak the rich cards, but this was the easiest fix 😉